### ISSUE TITLE
Consider changing thumbnail size back to 100px

### ISSUE THREAD

**Issue by alawvt** [VTUL/vtechworks#663]
Now that we are running DSpace 6, consider changing thumbnail size back to 100px. The original thumbnail settings were maxwidth=100px. A common size of these thumbnails was 99px × 128px, e.g. https://vtechworks.lib.vt.edu/handle/10919/45696.

Setting in https://github.com/VTUL/vtechworks/blob/46faa5183036b5625c43c3bea7bf18f058d3cc77/dspace/config/dspace.cfg#L888`.

```
# maximum width and height of generated thumbnails
thumbnail.maxwidth  = 80
thumbnail.maxheight = 80
```
[To run media-filter manually](https://wiki.lyrasis.org/display/DSDOC6x/Mediafilters+for+Transforming+DSpace+Content), 
> Identifier mode : [dspace]/bin/dspace filter-media -i 123456789/2
    Restrict processing to the community, collection, or item named by the identifier - by default, all bitstreams of all items in the repository are processed. The identifier must be a Handle, not a DB key. This option may be combined with any other option.

- [x] Run media-filter manually to create a thumbnail of an ETD at 80 px
- [x] Take a screenshot of the ETD in browse view.
- [x] Change thumbnail.maxwidth  = 100 and thumbnail.maxheight = 100 
- [x] Run media-filter manually to create a thumbnail of the same ETD at 100 px.
- [x] Take a screenshot of the ETD in browse view.
- [x] Post both screenshots.

The maximum thumbnail size should be 100 pixels wide and 100 pixels tall.

From #661, 
> I notice the text on the newer thumbnails is not as bold as the earlier thumbnails (pre-~2016). The new thumbnails are 80px wide. I think the old ones were 100px wide.

__________
From: Keith Gilbertson <keith.gilbertson@vt.edu>
Date: Wednesday, August 22, 2018 at 10:54 AM
To: Anne Lawrence <alaw@vt.edu>
Subject: Re: Thumbnail changes?

No, the thumbnail filters configured in DSpace don’t handle video files. The one’s we’ve uploaded would remain.

The “other custom thumbnails”, yes. Unless you have a list of the image thumbnails that we did manually (for example, Philip used a different page for one item because it looked better), then you could also utilize a skip flag in the media filter to skip regeneration of thumbnails for particular items.

_____________
On Wed, Aug 22, 2018 at 10:25 AM Anne Lawrence <alaw@vt.edu> wrote:

Keith,
 
Would that overwrite switch for thumbnails overwrite all our custom thumbnails for videos? I would hate to overwrite those and other custom thumbnails.
 
Philip,
 
I suggest we consider thumbnail size after moving to DSpace 6. Please feel free to add that as a task in the Backlog.
 
Thanks,
 
Anne
_________ 
From: Keith Gilbertson <keith.gilbertson@vt.edu>
Date: Wednesday, August 22, 2018 at 9:46 AM
To: Philip Young <pyoung1@vt.edu>
Cc: Anne Lawrence <alaw@vt.edu>
Subject: Re: Thumbnail changes?
 
We’ve had enough changes to thumbnails that I don’t even remember whether this was a decision that we made on purpose, or if it was done on accident. As Anne may have mentioned, we switched thumbnail builder plug-ins when we moved to DSpace 5 several years ago. Also, when we were upgrading to DSpace 4.2 in September of 2014, we switched to a Mirage based theme and also adjusted the size of the thumbnails then.
 
If you want the thumbnails to be a uniform size (except where unusual aspect ratios prevent that from happening), there’s a command line switch on the media filter that processes all items, and not just ones where derivatives (in this case, thumbnails) haven’t yet been generated. There’s also a switch to just run the media filter on a specific collection, which could be good for testing locally before trying in production. And I’m not suggesting that this is higher priority than what the group is already working on.
 
In addition to the actual size of the thumbnails, there may be code in the HTML or css for the pages to constrain thumbnails to a specific size. I don’t know if that’s the case anymore, but I seem to recall someone adding that in one theme, perhaps a previous theme that we no longer use. But it’s something to keep in mind if sizes still seem wrong after reconfiguring and rebuilding thumbnails on a test collection.
 
—keith
 
_________ 
 
On Wed, Aug 22, 2018 at 8:38 AM Philip Young <pyoung1@vt.edu> wrote:

Well, it looks much better, and doesn't seem to adversely affect the index.  Keith, do you know why thumbnail sizes have changed?  Thanks,

Philip

_________
On Wed, Aug 22, 2018 at 8:32 AM, Anne Lawrence <alaw@vt.edu> wrote:

Philip,
 
I looked at an item with a larger thumbnail, https://vtechworks.lib.vt.edu/handle/10919/47355?show=full, and see that it was loaded in 2014. I vaguely recall that we may have had the thumbnail size set then to 100 pixels wide, rather than 80 pixels wide. The setting has not changed recently, as far as I know.
 
Thanks,
 
Anne
___________
From: Philip Young <pyoung1@vt.edu>
Date: Wednesday, August 22, 2018 at 8:25 AM
To: Anne Lawrence <alaw@vt.edu>
Subject: Thumbnail changes?
 
Hi Anne,

Has our thumbnail process changed?  I noticed a significant variation in size when scrolling through the index of this collection: https://vtechworks.lib.vt.edu/handle/10919/24330.  Thanks,

Philip

**Comment by alawvt** [comment]
See https://wiki.lyrasis.org/display/DSDOC6x sections 1 and 2 to understand how to make changed to dspace.cfg.

**Comment by kdweeks** [comment]
Attached is the preview of a thumbnail at the maximum of 80x80 px and 100x100 px.
<img width="554" alt="Thumbnail_80x80" src="https://user-images.githubusercontent.com/473202/70824634-9df8cf80-1db0-11ea-94be-fb3a93b0fc3e.png">

<img width="554" alt="Thumbnail_100x100" src="https://user-images.githubusercontent.com/473202/70824660-a94bfb00-1db0-11ea-8962-86757982e34d.png">

**Comment by kdweeks** [comment]
I updated the local.cfg file and set the dspace.cfg file back to the default settings.   I reprovisioned the virtual machine and tested the thumbnail generation on a second file to make sure the settings were correct.  

<img width="771" alt="Thumbnail_2_100x100" src="https://user-images.githubusercontent.com/473202/70832575-74956f00-1dc3-11ea-996f-83262f78389a.png">

**Comment by kdweeks** [comment]
I did a final test uploading a custom size thumbnail to verify the generation size should be the size shown.  It seems there are no other constraining size limits for the thumbnail view.

<img width="782" alt="Thumbnail_custom" src="https://user-images.githubusercontent.com/473202/70833461-d060f780-1dc5-11ea-8db6-7eccfd91f39b.png">

**Comment by alawvt** [comment]
@kdweeks, thanks for determining that thumbnail.maxwidth and thumbnail.maxheight only affect the size of generated thumbnails and not the display size of thumbnails.

**Comment by alawvt** [comment]
Please create a pull request with this change.

**Comment by kdweeks** [comment]
This issue is resolved by #666

**Comment by alawvt** [comment]
New thumbnails will have `maxwidth=maxheight=100px`. This is a compromise of size for better viewing but avoiding excessive scrolling in browse view or search results. 

We plan to regenerate thumbnails for existing items so thumbnail sizes will be more consistant. The existing thumbnails are either `maxwidth=maxheight= 80px` or `maxwidth=100` with unlimited maxheight.
